cmd/utils.go

```diff
@@ -1491,13 +1495,17 @@ func GenServiceYaml(log *LoggingConfig, appName string, ports []string, pdir str
 	service.Spec.ServiceType = "NodePort"
 	service.Spec.Ports = make([]Port, len(ports))
```

Comment:
This is adding a empty port field when have a debug port, because inside the for loop we will skiping that port and not setting values: 
```
- port: 0
  targetPort: 0
```
Instead only add to this list instead of setting a fixed size.